multisig-wallet/project/multisig-contract/tests/functions/execute_transaction.rs

```diff
@@ -0,0 +1,39 @@
+mod success {
+
+    #[ignore]
+    #[tokio::test]
+    async fn executes_transaction() {
+        // Currently ignored and unimplemented, awaiting the ability to implement https://github.com/FuelLabs/sway-applications/issues/22
+    }
+}
+
+mod revert {
+
+    #[ignore]
+    #[tokio::test]
+    #[should_panic]
```

Comment:
I would also add the `expected = "Error"` to make it even clearer for future devs which line should fail.